tests/mock_web_api_server.py

```diff
@@ -253,9 +257,22 @@ def stop(self):
         self.join()
 
 
+class ReceivedRequestsHandler:
+    def __init__(self, queue: Queue):
+        self.queue = queue
+        self.received_requests = {}
+
+    def get(self, key, default=None):
+        while not self.queue.empty():
+            path = self.queue.get()
+            self.received_requests[path] = self.received_requests.get(path, 0) + 1
+        return self.received_requests.get(key, default)
+
+
 def setup_mock_web_api_server(test: TestCase):
     test.server_started = threading.Event()
-    test.thread = MockServerThread(test)
+    test.received_requests_handler = ReceivedRequestsHandler(Queue())
```

Comment:
For the same reason I mentioned above, `test.received_requests.get("/auth.test")` sounds more natural to me.

Reply:
Agree 💯